configure.py

```diff
@@ -1493,7 +1493,7 @@ def main():
   set_build_var(environ_cp, 'TF_NEED_KAFKA', 'Apache Kafka Platform',
                 'with_kafka_support', False, 'kafka')
   set_build_var(environ_cp, 'TF_ENABLE_XLA', 'XLA JIT', 'with_xla_support',
-                False, 'xla')
+                True, 'xla')
```

Comment:
@parallelo do we want to enable XLA by default, just like what we did in TF 1.3, or have it disabled by default?